engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/message/MessageCorrelationResultWithVariableDto.ftl

```diff
@@ -0,0 +1,35 @@
+<@lib.dto>
+
+    <@lib.property
+        name = "resultType"
+        type = "string"
+        desc = "Indicates if the message was correlated to a message start event or an 
+                intermediate message catching event. In the first case, the resultType is 
+                `ProcessDefinition` and otherwise `Execution`."/>
+
+    <@lib.property
+        name = "processInstance"
+        type = "object"
```

Comment:
Shouldn't this be of type `ref`?
```suggestion
        type = "ref"
```

Reply:
@yanavasileva, I'm not completely sure about this. Can you provide some input?

Reply:
I will try to come to this later today

Reply:
should be `ref`
currently, it's not full at the end file:
```
          "execution": {
            "type": "object",
            "description": "This property only has a value if the resultType is set to `Execution`.\nThe execution with the properties as described in the\n[get single execution](https://docs.camunda.org/manual/develop/reference/rest/execution/get/) method."
          },
```

Reply:
and the `ExecutionDto` is missing or I am missing something?

Reply:
This means we can not use descriptions in these cases, right? I think this is a problem, beacause ref does not respect sibling elements. They are ignored. 
https://swagger.io/docs/specification/using-ref/ ($ref and Sibling Elements)

Reply:
yep, ref cannot have a description on this level, they can have inside the dto
this guy cannot be put here:
> This property only has a value if the resultType is set to `Execution`